unit-tests/func/memory/test-extrinsics.cpp

```diff
@@ -0,0 +1,394 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+//#cmake: static!
+//#test:device D400*
```

Comment:
The test applies for L515 and SR300.
Does this label limits it to D400 only? Will it pass with L515/SR300?